ui/app/templates/vault/cluster/secrets/backend/list.hbs

```diff
@@ -57,8 +57,21 @@
 
   {{#if model.meta.total}}
     {{#each model as |item|}}
-      {{partial options.listItemPartial}}
-    {{else}}
+    {{!-- Because of the component helper cannot use glimmer nested SecretList::Item --}}
+      {{#let (component options.listItemPartial) as |Component|}}
```

Comment:
Very nice solution 👍 Not sure I totally understand the comment above, but I'm gathering it's justifying the `#let`?

Reply:
It's more about having the `options.listItemPartial` return `SecretList::Item` which is the new ember way of nested components, we have to use the old way of `secret-list/item` because of the `component` helper.  It was a TIL thing as I went down the rabbit hold of not understanding why it couldn't find my component.